packages/chrome-extension/src/internal/clerk.ts

```diff
@@ -22,19 +23,22 @@ export type CreateClerkClientOptions = {
   publishableKey: string;
   scope?: Scope;
   storageCache?: StorageCache;
-  syncSessionWithTab?: boolean;
+  syncHost?: string;
```

Comment:
We probably can't do anything else, but when trying to load this URL from a env var I had to do the following: 

```ts
const SYNC_HOST = process.env.PLASMO_PUBLIC_SYNC_HOST
syncHost={`http://${SYNC_HOST}`}`
```
The following resulted in a TS error of
```
 `Type 'string' is not assignable to type '`http://${string}\` | `https://${string}`'. `
```

```ts
const SYNC_HOST = process.env.PLASMO_PUBLIC_SYNC_HOST
syncHost={`http://${SYNC_HOST}`}
```

It is not a major issue -- I wrote the docs using the first version. If this can be improved it would be nice but not required.

Reply:
@royanger I can loosen the type here to just `string`

Reply:
I would say that's fine. Our docs will be clear on what to add. Or could we make it the following? Its a little loose, but at least the type info in the customer's editor should also show/imply that `http` or `https` is required.

```
`http://${string}\` | `https://${string}` | string
```

Either way, its not a blocker. I have the docs covering the current implementation. Choose the option you feel most comfortable with, and if its a change let me know so I can update the docs.